fsevents.cc

```diff
@@ -45,7 +46,7 @@ namespace fse {
 
     // Common
     CFArrayRef paths;
-    CFMutableArrayRef events;
+    std::vector<fse_event*> *events;
```

Comment:
I'd just make it a std::vector, not a pointer-to-std::vector.  That way you don't have to new and delete it in the constructor and destructor respectively.